Tasks/Buhayev/Task1/Task1.py

```diff
@@ -0,0 +1,85 @@
+#simplest implementation
+amount, duration, percent, factor = 20000, 5, 15, 12
+simplest_total_deposit = round(amount * ((1 + percent / 100 / factor) ** (duration * factor)),2)
```

Comment:
using int here is not wise, since it may lead to some errors, and the task is related to sensitive topic of money, so it's better to use decimal, -1 point for the unwise choice